src/utils/router_state.rs

```diff
@@ -204,22 +203,34 @@ pub fn get_node_by_id(id: &str) -> Result<&'static Node, String> {
 }
 
 /// Initialize the router with vertiports from the storage service
-pub fn init_router_from_vertiports(vertiports: &[Vertiport]) {
+pub fn init_router_from_vertiports(vertiports: &[Vertiport]) -> Result<(), String> {
     let nodes = vertiports
         .iter()
         .map(|vertiport| Node {
             uid: vertiport.id.clone(),
             location: Location {
-                latitude: OrderedFloat(vertiport.data.as_ref().unwrap().latitude),
-                longitude: OrderedFloat(vertiport.data.as_ref().unwrap().longitude),
+                latitude: OrderedFloat(
+                    vertiport
+                        .data
+                        .as_ref()
+                        .expect("Vertiport data not set")
+                        .latitude,
+                ),
+                longitude: OrderedFloat(
+                    vertiport
+                        .data
+                        .as_ref()
+                        .expect("Vertiport data not set")
```

Comment:
Make the expect message more explicit? Like "Something went wrong when parsing latitude data of vertiport id: ${vertiport.id}"

Reply:
added message